algorithms/kernel/multiclassclassifier/multiclassclassifier_train_batch_container.h

```diff
@@ -58,16 +58,18 @@ services::Status BatchContainer<algorithmFPType, method, cpu>::compute()
     const classifier::training::Input * input = static_cast<const classifier::training::Input *>(_in);
     Result * result                           = static_cast<Result *>(_res);
 
-    const NumericTable * a[2];
-    a[0]                              = static_cast<NumericTable *>(input->get(classifier::training::data).get());
-    a[1]                              = static_cast<NumericTable *>(input->get(classifier::training::labels).get());
+    const NumericTable * a[3];
+    a[0] = static_cast<NumericTable *>(input->get(classifier::training::data).get());
+    a[1] = static_cast<NumericTable *>(input->get(classifier::training::labels).get());
+    a[2] = static_cast<NumericTable *>(input->get(classifier::training::weights).get());
```

Comment:
I didn't noticed that size of weights input table was checked in Input::check().
Please add appropriate check of input size. Seems it should be:
`input->get(classifier::training::data)->getNumberOfRows() == input->get(classifier::training::weights)->getNumberOfRows()`

Reply:
Yes, It checked here: https://github.com/oneapi-src/oneDAL/blob/master/algorithms/kernel/classifier/classifier_train.cpp#L107